File: Posts/Problems-And-Solutions/Categories/Prime-Number/N-th-Prime/solution-brute.cpp
#include <bits/stdc++.h>

using namespace std;

bool is_prime(int n) {
    if (n < 2) {
        return false;
    }

    if (n == 2 || n == 3) {
        return true;
    }

    if (n % 2 == 0) {
        return false;
    }

    for (int i = 3; i <= sqrt(n); i += 2) {
        if (n % i == 0) {
            return false;
        }
    }

    return true;
}

int main() {
	int queries;
    cin >> queries;
    while (queries--) {
        int n, p = 1, counter = 0;
        cin >> n;
        while (counter != n) {
            ++p;
            if (is_prime(p) == true) {
                ++counter;
            }
        }
        cout << p << endl;
    }

    return 0;
}

File: Posts/Problems-And-Solutions/Categories/Prime-Number/N-th-Prime/solution-brute.py
import math


def is_prime(n: int):
    if (n < 2):
        return False
    
    if (n == 2 or n == 3):
        return True
    
    if (n % 2 == 0):
        return False
    
    for i in range(3, int(math.sqrt(n)) + 1, 2):
        if (n % i == 0):
            return False

    return True


queries = int(input())

while (queries):
    queries -= 1
    
    n = int(input())
    p = 1
    counter = 0
    
    while (counter != n):
        p += 1
        if (is_prime(p) == True):
            counter += 1
    
    print(p)